src/SmartComponents/Remediations/RunStatus.js

```diff
@@ -40,56 +38,25 @@ const renderStatusIcon = (status) => ({
         aria-label="Remediation failed" />
 })[status];
 
-const RunStatus = ({ id, name, index }) => {
-    const [playbookRun, setPlaybookRun] = useState({});
-    const [hasData, setHasData] = useState();
-    const [loaded, setLoaded] = useState();
-    const [isCriticalError, setIsCriticalError] = useState(false);
-
+const RunStatus = ({ id, name, index, playbookRuns }) => {
+    const hasData = playbookRuns ? playbookRuns.data?.length > 0 : false;
     const intl = useIntl();
 
-    useEffect(() => {
-        const fetchPlaybookRun = async () => {
-            try {
-                const response = await API.get(`${REMEDIATIONS_PLAYBOOK_RUNS_FETCH_URL}${id}/playbook_runs?limit=1&offset=0&sort=-updated_at`);
-                setPlaybookRun(response.data.data);
-                setLoaded(true);
-                setHasData(response.data.data.length);
-            } catch (error) {
-                setLoaded(false);
-                setHasData(false);
-                setIsCriticalError(true);
-                // eslint-disable-next-line no-console
-                console.error('Error contacting remediations API');
-            }
-        };
-
-        fetchPlaybookRun();
-    }, [id]);
-
     return <div className="insd-c-remediations-container">
         <div className="insd-c-remediation__status">
-            { loaded === undefined && <Skeleton size='md' /> }
-            { (loaded && hasData) ?
+            { hasData ?
                 <React.Fragment>
-                    {renderStatusIcon(normalizeStatus(playbookRun[0].status))}
+                    {renderStatusIcon(normalizeStatus(playbookRuns.data[0].status))}
                     <p className='insd-c-remediation__status-text'>
-                        { intl.formatMessage(messages.remediationsPlaybookStatus, { status: normalizeStatus(playbookRun[0].status) }) }
+                        { intl.formatMessage(messages.remediationsPlaybookStatus, { status: normalizeStatus(playbookRuns.data[0].status) }) }
                     </p>
                 </React.Fragment>
-                : null
-            }
-            { loaded && !hasData &&
+                :
                 <React.Fragment>
                     <QuestionIcon />
                     <p>{intl.formatMessage(messages.remediationsPlaybookNoActivity)}</p>
                 </React.Fragment>
             }
-            { isCriticalError &&
```

Comment:
I think this is an error-handling message and we should keep it.

Reply:
This error only happened when downloading the playbook_runs separately.  Now that the playbook_runs are integrated into REMEDIATIONS_FETCH_URL we don't need this error handling anymore.